src/SFML/CMakeLists.txt

```diff
@@ -21,14 +21,37 @@ if(CSFML_LINK_SFML_STATICALLY)
     set(SFML_STATIC_LIBRARIES TRUE)
     add_definitions(-DSFML_STATIC)
 endif()
-find_package(SFML 2.6 COMPONENTS network graphics audio REQUIRED)
+set(SFML_MODULES "system")
+if(CSFML_BUILD_AUDIO)
+    list(PREPEND SFML_MODULES "audio")
+endif()
+if(CSFML_BUILD_GRAPHICS)
+    list(PREPEND SFML_MODULES "graphics")
+endif()
+if(CSFML_BUILD_NETWORK)
+    list(PREPEND SFML_MODULES "network")
+endif()
+find_package(SFML 2.6 COMPONENTS ${SFML_MODULES} REQUIRED)
```

Comment:
Missing instructions for the Window module.

Reply:
See #250